ChronoKeeper/StoryChunkExtractorRDMA.cpp

```diff
@@ -53,7 +52,7 @@ int chronolog::StoryChunkExtractorRDMA::processStoryChunk(StoryChunk*story_chunk
         LOG_DEBUG("[StoryChunkExtractorRDMA] Serialized story chunk size: {}", serialized_story_chunk_size);
 
         std::vector <std::pair <void*, std::size_t>> segments(1);
-        segments[0].first = (void*)(serialized_buf);
+        segments[0].first = (void*)(&serialized_buf[0]);
         segments[0].second = serialized_story_chunk_size + 1;
```

Comment:
is line 46 a reason for serialized _story_chunk_size+1 ?

Reply:
`serialized_story_chunk_size+1` is copied from Thallium's example code. Yes, line 46 is for this.